#!/usr/bin/env python

PACKAGE="fre_row_navigation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       name,                type,   level, description,           default, min, max

gen.add("p_gain",            double_t, 0,  "p gain for turning: ", 1.0,    0.0, 100.0)
gen.add("i_gain",            double_t, 0,  "i gain for turning: ", 0.0,    0.0, 0.5)
gen.add("d_gain",            double_t, 0,  "d gain for turning: ", 0,    0.0, 0.5)

gen.add("start", bool_t,   0, "start: ",   False)

exit(gen.generate(PACKAGE, "PidTester", "PidTester"))
